package.json

```diff
@@ -90,11 +91,13 @@
     "mocha": "^5.2.0",
     "monaco-editor": "^0.15.6",
     "ngx-autosize-input": "^2.2.2",
+    "ngx-markdown": "^7.1.4",
```

Comment:
You just added it to package json? I didn't find any usage of the library

Reply:
Yeah, I couldn't make it work.
Removed